docs/source/getting-started.rst

```diff
@@ -280,8 +280,8 @@ commands which provide basic functionality. (More on that later.) For now, thoug
     package com.example.helloworld;
 
     import io.dropwizard.Application;
-    import io.dropwizard.setup.Bootstrap;
-    import io.dropwizard.setup.Environment;
+    import Bootstrap;
```

Comment:
Overall looks good, but some doc imports seem to be broken

Reply:
@zUniQueX Thanks for noticing! 😄